src/components/App.jsx

```diff
@@ -9,7 +9,7 @@ import SiteFooter from '@edx/frontend-component-footer';
 import { fetchUserAccount, UserAccountApiService } from '@edx/frontend-auth';
 
 import apiClient from '../config/apiClient';
-import { getLocale, getMessages } from '../i18n/i18n-loader';
+import { getLocale, getMessages } from '@edx/frontend-i18n'; // eslint-disable-line
```

Comment:
What does it complain about?

Reply:
eslint can't resolve the import properly. To get it to work you need to add some packages like this one: https://www.npmjs.com/package/eslint-import-resolver-alias

I figured it wasn't worth the effort.